crates/egui_kittest/src/snapshot.rs

```diff
@@ -155,6 +155,10 @@ impl Display for SnapshotError {
     }
 }
 
+/// If this is set, we update the snapshots (if different),
+/// and _succeed_ the test.
+/// This is so that you can set `UPDATE_SNAPSHOTS=true` and update _all_ tests,
+/// without `cargo test` failing on the first failing crate.
 fn should_update_snapshots() -> bool {
     std::env::var("UPDATE_SNAPSHOTS").is_ok()
```

Comment:
It would be nice if we actually checked for a truth value here. I did `UPDATE_SNAPSHOTS=0` once and it didn't do what I expected.